src/cljs_tooling/complete.clj

```diff
@@ -188,12 +189,12 @@
   [candidate prefix]
   (.startsWith ^String (:candidate candidate) (str prefix)))
 
-(defn- enrich-candidate [candidate env {:keys [extra-metadata]}]
+(defn- enrich-candidate [candidate env {:keys [context-ns extra-metadata]}]
   (if (seq extra-metadata)
-    (let [var-meta (a/find-var env (symbol (str (:ns candidate)) (:candidate candidate)))]
+    (let [var-meta (i/info env (symbol (str (:ns candidate)) (:candidate candidate)) context-ns)]
       (cond-> candidate
         (and (:arglists extra-metadata) (:arglists var-meta))
-        (assoc :arglists (apply list (map pr-str (eval (:arglists var-meta)))))
+        (assoc :arglists (apply list (map pr-str (i/unquote-1 (:arglists var-meta)))))
```

Comment:
Not sure we need to unquote again, since `info` calls `format-var` which already does the unquoting: https://github.com/gtrak/cljs-tooling/blob/master/src/cljs_tooling/info.clj#L31-L37